skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md

```diff
@@ -90,7 +97,7 @@ Accept wildcard characters: False
 ```
 
 ### -Global
-When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant. To skip a warning when you do this operation, specify "-Global".
+When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant, except any that have an explicit policy assignment. To skip a warning when you do this operation, specify this parameter.
```

Comment:
```suggestion
When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant, except anyone that have an explicit policy assignment. To skip a warning when you do this operation, specify this parameter.
```

Reply:
'anyone that have' to 'anyone that has'.

Reply:
Are you sure about this update? It's the output of the cmdlet:
![image](https://user-images.githubusercontent.com/33589238/80093688-57279d00-853b-11ea-9dd4-0ef3447bf964.png)

I think it's related to "any" user, "anyone" user doesn't look good.